app/helpers/maintenance_tasks/tasks_helper.rb

```diff
@@ -103,6 +103,16 @@ def csv_file_download_path(run)
 
     # Return the appropriate field tag for the parameter
     def parameter_field(form_builder, parameter_name)
+      # If the parameter has a `validates_inclusion_in` parameter,
+      # generate a dropdown list of options
```

Comment:
```suggestion
      # If the parameter has a `validates_inclusion_of` validator,
      # generate a dropdown list of options
```
Nit, can we also move this above the method definition so it becomes part of the docs?